web/templatetags/timezone_filters.py

```
@@ -0,0 +1,75 @@
+    if not is_aware(value):
+        try:
+            value = make_aware(value)
+        except (ValueError, TypeError):
+            # If we can't make it aware, just return it as is
+            return value
+
+    return localtime(value)
+
+
+@register.filter(name="localtime_format")
+def localtime_format(value, format_string=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `localtime_format`

(ANN201)